body.tex

```diff
@@ -203,22 +260,23 @@ \section{Long Term Planning}
 \end{description}
 
 Some of these milestones are exposed to external reviewers (usually reserved for Level 1 milestones): it is important that these be delivered on time and to specification.
-Level 1 and 2 milestones are under change control once they are defined and described in a \gls{JIRA} Milestone issue.
+Level 1 and 2 milestones are under change control once they are defined and described in a \gls{JIRA} Milestone issue. Note the change control process is under development as a Pre-Operations activity. 
+
 Level 3 milestones are defined for use within Departments and not required to go under project change control, but properly adhering to the plan is important: your colleagues in other teams will use these milestones to align their schedules with yours, so they rely on you to be accurate.
 
-Epics should work to achieve milestones i.e. they may be blocking issues on the milestones.
+Epics should work to achieve milestones, i.e., they may be blocking issues on the milestones.
 When a detailed description of work for a given \gls{epic} is known, it is described in \gls{JIRA}.
 It should then be assigned to the appropriate \glspl{cycle}.
 
-Progress is tracked toward achieving milestones in Smartsheet by monitoring completed story points in \gls{JIRA} epics and rolling up the total progress.
+Progress is tracked toward achieving milestones in Smartsheet by monitoring completed story points on linked issues in \gls{JIRA} epics and rolling up the total progress.
 To ensure success, \gls{JIRA} epics must be completely detailed out prior to a full 6-month cycle and total effort should be estimated out for an entire fiscal year of effort, as detailed below.
```

Comment:
and all milestones should appear in Jira which is the source of truth.

Reply:
```suggestion
To ensure success, \gls{JIRA} epics must be completely detailed out prior to a full 6-month cycle and total effort should be estimated out for an entire fiscal year of effort, as detailed below.
All milestones should appear in \gls{JIRA} with a milestone issue type as the source of truth. 
```